### PR TITLE
Create tag for the origins mod

### PR DESCRIPTION
**Pull request by OroArmor**
With the sudden popularity of the origins mod, a dedicated tag pointing people toward the official discord server should be added

### CODE REVIEW

File: tags/faq/origins.ytag [hunk 1]
```diff
@@ -0,0 +1,5 @@
+type: text
+
+---
+
+For more specific help with the Origins mod, go to the dedicated Origins discord server: https://discord.gg/4mTMHu3
```
Comment (Platymemo) [inline tags/faq/origins.ytag:5]: Possibly a little terse? maybe `please go to` or `you can go to` would be better.

**Review by Platymemo — COMMENTED** [review]
It looks good and I think it is needed, at least temporarily with the new surge in popularity of the mod, but the wording is a little terse. I left a comment on it.

**Review by liach — APPROVED** [review]

**Review by Technici4n — APPROVED** [review]

**Review by LambdAurora — APPROVED** [review]

**Review by Dolphin2Point1 — APPROVED** [review]

**Comment by YTG1234** [comment]
Why not just alias to the community Discords tag?

**Review by yitzy299 — APPROVED** [review]
Looks good

**Comment by OroArmor** [comment]
> Why not just alias to the community Discords tag? This could serve when more mods get popular as well.
> 
> Also, should be `discord/origins` instead of `faq/origins`.

The reason I don't think it should alias to the community discords is that people can be lazy, and providing a direct way to join is best

I will move it to the discord section though

**Comment by Technici4n** [comment]
Yes we need this, someone just asked again.

**Review by yitzy299 — APPROVED** [review]

**Pull request closed without merging**